hail/python/hail/matrixtable.py

```diff
@@ -1366,8 +1366,8 @@ def semi_join_rows(self, other: 'Table') -> 'MatrixTable':
         -----
         The row key type of the matrix table must match the key type of `other`.
 
-        This method does not change the schema of the matrix table; it is a
-        filtering the matrix table to row keys not present in another table.
+        This method does not change the schema of the matrix table; it is 
```

Comment:
Your PR is failing style checks, remove the whitespace after the `is` on this line.